includes/class-federated-reactions-settings.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Federated Reactions Settings file.
+ *
+ * @package ActivityPub
+ */
+
+namespace Activitypub;
```

Comment:
I would put them in a subfolder and use the namespace `Activitypub\Settings` or similar. Then we could migrate the rest of the settings over time.

Reply:
and do we have to use `federated` in the filename?

Reply:
Fixed the filename in [c5d6551](https://github.com/Automattic/wordpress-activitypub/pull/1070/commits/c5d65516b52b0684472f71d6de8701967e4c41b2).

Reply:
> I would put the file in a subfolder and use the namespace `Activitypub\Settings` or similar. Then we could migrate the rest of the settings over time.

Agreed! I'd even go a step further and create an admin subfolder and have Settings as part of that, to bundle all admin-related functionality. Let's do that as a follow-up?